meson.build

```diff
@@ -10,6 +10,16 @@ project('nntrainer', 'c', 'cpp',
     'buildtype=release'
   ]
 )
+
+# Set version info
+nntrainer_version = meson.project_version()
+nntrainer_version_split = nntrainer_version.split('.')
+
+add_project_arguments('-DVERSION="' + nntrainer_version + '"', language: ['c', 'cpp'])
+add_project_arguments('-DVERSION_MAJOR=' + nntrainer_version_split[0], language: ['c', 'cpp'])
```

Comment:
`NNTRAINER_VERSION_MAJOR`  or `VERSION_MAJOR` ?

Reply:
The version information is stored as an argument in meson.build in the root.
However, in api/ccapi/meson.build, we define variables to add to the header.
It appears that the values defined in meson.build in the root cannot be used when creating headers.

project arguments
- add_project_arguments('-DVERSION_MAJOR=' + nntrainer_version_split[0], language: ['c', 'cpp'])

Definition for creating header
- version_conf.set('__NNTRAINER_VERSION_MAJOR__', nntrainer_version_split[0])

Reply:
> It appears that the values defined in meson.build in the root cannot be used when creating headers.

But,, project arguments are global var.  please check scope

```
Project arguments
Project arguments work similar to global arguments except that they are valid only within the current subproject. The usage is simple:

add_project_arguments('-DMYPROJ=projname', language : 'c')
This would add the compiler flags to all C sources in the current project.
```

Reply:
Modified to use only one.

In the case of andoird, separate settings were required, so it was added.